frontend/src/global_styles/content/_autocomplete.sass

```diff
@@ -112,7 +112,7 @@ div.autocomplete
   border-color: var(--borderColor-default) !important
 
 .ng-select-container
-  background-color: transparent !important
+  background-color: var(--body-background) !important
```

Comment:
Because of this change, the select fields are now again darker than the other input fields

<img width="502" alt="Bildschirmfoto 2024-07-04 um 08 43 21" src="https://github.com/opf/openproject/assets/7457313/bfc2f19b-d6a7-4b62-a703-a7e6a05be8cb">